src/DotNet.Status.Web/Startup.cs

```diff
@@ -188,10 +200,10 @@ private void AddServices(IServiceCollection services)
             services.AddScoped<SimpleSigninMiddleware>();
             services.AddGitHubTokenProvider();
             services.AddSingleton<IInstallationLookup, InMemoryCacheInstallationLookup>();
-            services.AddContextAwareAuthenticationScheme(o =>
-            {
-                o.SelectScheme = p => p.StartsWithSegments("/api") ? "github-token" : IdentityConstants.ApplicationScheme;
-            });
+            //services.AddContextAwareAuthenticationScheme(o =>
```

Comment:
Keeping commented out code?

Reply:
:hang_head_low_emoji: No... I saw that several times and thought "I should deleted that"...

Reply:
Done!